### PR TITLE
Fix duplicated values for matric group

### PR DESCRIPTION
**Pull request by RomanBats**
Closes https://github.com/dotnet/dotnet-monitor/issues/2021

### CODE REVIEW

**Comment by wiktork** [comment]
@RomanBats Great catch! For the fix though, you will need to pass each metric through normalization and then output the normalized value. As an example, `working set` is originally in MB, but then gets converted to bytes. Ideally we would convert `PrometheusDataModel.Normalize` into two functions, one that handles value normalization and another that handles everything else.

**Review by wiktork — CHANGES_REQUESTED** [review]
See other comment for requested changes.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-monitor?pullRequest=2149) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-monitor?pullRequest=2149) before we can accept your contribution.<br/><br/>:x: RomanBats [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-monitor?pullRequest=2149)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-monitor?pullRequest=2149) it.</sub>

File: src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.UnitTests
+{
+    public class PrometheusDataModelTests
+    {
+        [Theory()]
```
Comment (wiktork) [inline src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs:11]: Thank you for expanding our tests.

File: src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        [InlineData("System.Runtime", "gc-heap-size", "MB", "systemruntime_gc_heap_size_bytes")]
+        [InlineData("System.Runtime", "gen-0-gc-count", "count", "systemruntime_gen_0_gc_count")]
+        [InlineData("System.Runtime", "gen-1-gc-count", "count", "systemruntime_gen_1_gc_count")]
+        [InlineData("System.Runtime", "gen-2-gc-count", "count", "systemruntime_gen_2_gc_count")]
+        [InlineData("System.Runtime", "threadpool-thread-count", "", "systemruntime_threadpool_thread_count")]
+        [InlineData("System.Runtime", "monitor-lock-contention-count", "count", "systemruntime_monitor_lock_contention_count")]
+        [InlineData("System.Runtime", "threadpool-queue-length", "", "systemruntime_threadpool_queue_length")]
+        [InlineData("System.Runtime", "threadpool-completed-items-count", "count", "systemruntime_threadpool_completed_items_count")]
+        [InlineData("System.Runtime", "alloc-rate", "B", "systemruntime_alloc_rate_bytes")]
+        [InlineData("System.Runtime", "active-timer-count", "", "systemruntime_active_timer_count")]
```
Comment (wiktork) [inline src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs:23]: I see some count values have a count unit and others are empty. While this is functionally the same, are you using the original metric to decide this or some other criteria?

**Review by wiktork — APPROVED** [review]

File: src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs (hunk 2 above)
Comment (RomanBats) [reply src/Tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/PrometheusDataModelTests.cs:23]: Yep, I got the list of default metrics from dotnet 6 web API

**Comment by RomanBats** [comment]
@wiktork, could you please tell me how can I restart checks for the PR?
It looks like it has failed due to timeout.

**Review by jander-msft — APPROVED** [review]

**Pull request merged**